cmd/rook/ceph/osd.go

```diff
@@ -286,17 +284,17 @@ func commonOSDInit(cmd *cobra.Command) {
 	clusterInfo.Monitors = mon.ParseMonEndpoints(cfg.monEndpoints)
 }
 
-func getLocation(clientset kubernetes.Interface, location string, topologyAware bool) (string, error) {
+func getLocation(clientset kubernetes.Interface, location string, pvcBacked bool) (string, error) {
 	locArgs, err := client.FormatLocation(cfg.location, cfg.nodeName)
 	if err != nil {
 		return "", fmt.Errorf("invalid location. %+v", err)
 	}
 	// use zone/region/hostname labels in the crushmap
-	if cfg.topologyAware {
+	if cfg.pvcBacked {
```

Comment:
Wait, i thought we were always going to add the topology info? If we're only doing it for pvcBacked, we're skipping other scenarios.